monitoring/healthz/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:16.18-slim
```

Comment:
@phelpsdb can we make sure to use node 18 here? @sliptype has put in a lot of hard work to get us there!